Makefile

```diff
@@ -1,6 +1,6 @@
 PWD := $(shell pwd)
 KVERSION    := $(shell uname -r)
-KDIR        := /lib/modules/$(KVERSION)/build
+KERNEL_SRC  ?= /lib/modules/$(KVERSION)/build
```

Comment:
Keep KDIR as it is documented here and in the kernel doc. Just do something like KERNEL_SRC ?= $(KDIR)

Reply:
You might need something different actually. If KERNEL_SRC is defined, then use it to redefine KDIR.

Reply:
I didn't know that the KDIR is a standard variable for the kernel source dir, but now I do a check if the variable KERNEL_SRC is set and assign it to KDIR. Otherwise use the old behavior.